Packs/Base/Scripts/GetErrors/GetErrors.yml

```diff
@@ -0,0 +1,27 @@
+commonfields:
+  id: GetErrors
+  version: -1
+name: GetErrors
+script: ''
+type: python
+tags:
+- Utility
+comment: Get the error(s) associated with a given entry/entries. Use ${lastCompletedTaskEntries} to check the previous task entries. The automation will return an array of the error contents from those entries.
+enabled: true
+args:
+- name: entry_id
+  required: true
```

Comment:
if you can get the lastCompletedTaskEntries from demisto.context(), then this arg should not be required

Reply:
I'll verify and update accordingly.